container-engine-lib/lib/backend_interface/objects/service/service_config.go

```diff
@@ -28,7 +30,11 @@ type privateServiceConfig struct {
 	// Configuration for container engine to pull an in a private registry behind authentication
 	// If nil, we will use the ContainerImageName and not use any auth
 	// Mutually exclusive from ImageBuildSpec, ContainerImageName
+
 	ImagerRegistrySpec *image_registry_spec.ImageRegistrySpec
+	// Reference to a image file that can be loaded into a backend image
+	// This is one of the options to reference a image and this value is optional (i.e. nil)
+	ImageLoad *image_load.ImageLoad
```

Comment:
Is there a usecase for this? I'd imagine users would be either a) specifying an existing Docker image, or b) getting an image through indirect means (either `ImageBuildSpec` or `NixBuildSpec`)